src/components/AssetsPage/AssetsPage.test.jsx

```diff
@@ -319,72 +325,6 @@ describe('<AssetsPage />', () => {
       });
     });
   });
-  describe('getAssets', () => {
```

Comment:
I removed the mapDispatchToProps tests and the request state tests, since the action creators are now thunks (much harder to test) and request state has been removed from Redux.